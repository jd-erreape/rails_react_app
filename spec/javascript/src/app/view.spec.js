import React from 'react';
import { shallow } from 'enzyme';
import { Navbar } from 'react-bootstrap';
import View from 'src/app/view';  

describe('View component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<View />);

    expect(wrapper).toMatchSnapshot();
  });

  it('contains the proper title', () => {
    const wrapper = shallow(<View />);

    expect(wrapper.find(Navbar.Brand).text()).toBe('Simple Rails / React App')
  });
});