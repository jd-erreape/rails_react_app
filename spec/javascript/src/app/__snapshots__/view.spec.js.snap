// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View component matches snapshot 1`] = `
<Fragment>
  <Navbar
    bg="light"
    collapseOnSelect={false}
    expand="lg"
    variant="light"
  >
    <Container
      fluid={false}
    >
      <NavbarBrand
        href="#home"
      >
        Simple Rails / React App
      </NavbarBrand>
    </Container>
  </Navbar>
  <Container
    fluid={false}
  >
    <Row>
      <Col
        sm={8}
      >
        <_default />
      </Col>
      <Col
        sm={4}
      >
        sm=4
      </Col>
    </Row>
  </Container>
</Fragment>
`;
